gamestonk_terminal/portfolio/port_controller.py

```diff
@@ -188,6 +203,42 @@ def call_dghold(self, _):
             print(e)
             print("")
 
+    def call_dgtopnews(self, _):
+        try:
+            dg_api.top_news_preview()
```

Comment:
As mentioned above, if you have that try-catch block inside function you don't need it here